android-patches/patches-droid-office-grouped/OfficeRNHost/React/CxxBridge/RCTCxxBridge.mm

```diff
@@ -1,10 +1,10 @@
---- "e:\\github\\fb-react-native-forpatch-base\\React\\CxxBridge\\RCTCxxBridge.mm"	2020-01-30 13:55:48.476581100 -0800
-+++ "e:\\github\\ms-react-native-forpatch\\React\\CxxBridge\\RCTCxxBridge.mm"	2020-02-14 10:59:16.805390300 -0800
-@@ -596,6 +596,7 @@
-     // This is async, but any calls into JS are blocked by the m_syncReady CV in Instance
-   _reactInstance->initializeBridge(
-       std::make_unique<RCTInstanceCallback>(self),
-+      nullptr, // Use default executor delegate
-       executorFactory,
-       _jsMessageThread,
-       [self _buildModuleRegistryUnlocked]);
+--- "e:\\github\\fb-react-native-forpatch-base\\React\\CxxBridge\\RCTCxxBridge.mm"	2020-01-30 13:55:48.476581100 -0800
```

Comment:
this can go away now that your other change has been merged to master? (thanks for fixing!)

Reply:
I reverted back to master but my machine automatically changes this file. Since this change is already in this is innocuous- the merge will just squash it and it won't do anything

Reply:
Can just remove from this PR? But that there is still a problem after pulling master suggests this is a larger problem. Regardless, something outside of the scope of this PR, but ideal if we can prevent this diff from making this PR easy to understand

Reply:
I'm having trouble removing it from this PR, but maybe there's some easy git way to remove it that I just don't know about... I tried a couple things but every time I get an older version to push up it automatically changes the line endings